sdks/python/container/run_validatescontainer.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+#
+#    Licensed to the Apache Software Foundation (ASF) under one or more
+#    contributor license agreements.  See the NOTICE file distributed with
+#    this work for additional information regarding copyright ownership.
+#    The ASF licenses this file to You under the Apache License, Version 2.0
+#    (the "License"); you may not use this file except in compliance with
+#    the License.  You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+# This script will be run by Jenkins as a post commit test. In order to run
+# locally make the following changes:
+#
+# LOCAL_PATH   -> Path of tox and virtualenv if you have them already installed.
+# GCS_LOCATION -> Temporary location to use for service tests.
+# PROJECT      -> Project name to use for dataflow and docker images.
+#
+# Execute from the root of the repository: sdks/python/container/run_validatescontainer.sh
+
+set -e
+set -v
+
+# pip install --user installation location.
+LOCAL_PATH=$HOME/.local/bin/
+
+# Where to store integration test outputs.
+GCS_LOCATION=gs://temp-storage-for-end-to-end-tests
+
+# Project for the container and integration test
+PROJECT=apache-beam-testing
+
+# Verify in the root of the repository
+test -d sdks/python/container
+
+# Verify docker and gcloud commands exist
+command -v docker
+command -v gcloud
+
+# Build the container
+DATE=$(date +%Y%m%d-%H%M%S)
+CONTAINER=gcr.io/$PROJECT/$USER-$DATE/python
+echo "Using container $CONTAINER"
+mvn clean install -DskipTests -Pbuild-containers --projects sdks/python/container -amd -Ddocker-repository-root=gcr.io/$PROJECT/$USER-$DATE -amd
```

Comment:
nit: "-amd" is included twice.

Small suggestion: it is more natural to use the date (and/or commit hash, perhaps) as the "tag" of the image instead of in the middle of the name: gcr.io/$PROJECT/$USER/python:$TAG. You can use "-Ddockerfile.tag=<tag>" to set the tag in the command as per sdks/CONTAINERS.md.

Reply:
Done. Removed the extra -amd and used gcr.io/$PROJECT/$USER/python:$TAG.